funnel/models/contact_exchange.py

```diff
@@ -163,9 +195,13 @@ def contacts_for_project_and_date(cls, user, project, date, archived=False):
             # because `project` may be an instance of ProjectId returned by `grouped_counts_for`
             Participant.project_id == project.id,
             db.cast(
-                db.func.date_trunc('day', db.func.timezone(project.timezone.zone, cls.scanned_at)),
-                db.Date) == date
+                db.func.date_trunc(
+                    'day', db.func.timezone(project.timezone.zone, cls.scanned_at)
+                ),
+                db.Date,
             )
+            == date,
```

Comment:
Is this fine?

Reply:
Yes. Binary operators can be on a line of their own. Black compares bytecode of pre- and post-format to confirm there are no syntactical changes.